client/app/components/QueryEditor.jsx

```diff
@@ -40,6 +40,9 @@ function buildKeywordsFromSchema(schema) {
   schema.forEach((table) => {
     keywords[table.name] = 'Table';
     table.columns.forEach((c) => {
+      if (c.length > 0) {
+        c = c.split('(')[0];
```

Comment:
If we send this upstream we'll probably want to do something different from pushing the column type info into this string and stripping it back out again like this.

Reply:
Oh, that's a feature that hasn't been upstreamed since it was denied there. Instead the schema browser rewrite has been started, e.g. @emtwo's schema drawer.

Reply:
In other words, this is just a fix for https://github.com/mozilla/redash/commit/17290cb738b308f759f1bcdea5a45f586de4a38f until we get the schema drawer functionality.